griptape/drivers/sql/sql_driver.py

```diff
@@ -46,6 +49,6 @@ def get_table_schema(self, table: str, schema: Optional[str] = None) -> Optional
             table = sqlalchemy.Table(
                 table, sqlalchemy.MetaData(bind=self.engine), schema=schema, autoload=True, autoload_with=self.engine
             )
-            return str([(c.name, c.type) for c in table.columns])
+            return str([(c.name, c.type) for c in getattr(table, "columns")])
```

Comment:
Same comment as above?

Reply:
same error as above

Reply:
Let's try what I suggested in the other comment.